packages/libsvm/libsvm.0.9.3/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "1.2"
+maintainer: "Oliver Gu <gu.oliver@yahoo.com>"
+authors: [ "Oliver Gu <gu.oliver@yahoo.com>"
+           "Dominik Brugger <dominikbrugger@fastmail.fm>" ]
+license: "LGPL-2.1+ with OCaml linking exception"
+homepage: "https://bitbucket.org/ogu/libsvm-ocaml"
+dev-repo: "https://bitbucket.org/ogu/libsvm-ocaml.git"
+bug-reports: "https://bitbucket.org/ogu/libsvm-ocaml/issues"
+tags: [ "clib:svm"  ]
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
+  ["ocamlfind" "remove" "libsvm"]
+]
+build-test: [
+  ["ocaml" "setup.ml" "-configure" "--enable-tests"]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-test"]
+]
+build-doc: [ "ocaml" "setup.ml" "-doc" ]
+depends: [
+  "base-threads"
+  "core_kernel" {>= "113.24.00"}
+  "lacaml" {>= "8.0.6"}
+  "ocamlbuild" {build}
+  "ocamlfind" {build & >= "1.3.1"}
+]
+depopts: [
+  "archimedes"
+  "core"
+  "gsl"
+  "ppx_sexp_conv"
+]
+conflicts: [
+  "core" {> "108.00.02"}
+  "core" {< "113.00.00"}
+  "ppx_sexp_conv" {> "113.09.00"}
+  "ppx_sexp_conv" {< "113.09.00"}
```

Comment:
These `ppx_sexp_conv` conflicts prevent installation because a different version of `ppx_sexp_conv` is required transitively by `core_kernel` in your `depends`.